src/elements/filters/FeColorMatrix.tsx

```diff
@@ -6,13 +6,13 @@ import {
   extractFilter,
   extractIn,
 } from '../../lib/extract/extractFilter';
-import { FilterColorMatrixType } from '../../lib/extract/types';
+import { FilterColorMatrixType, NumberArray } from '../../lib/extract/types';
 import FilterPrimitive from './FilterPrimitive';
 
 export type FeColorMatrixProps = {
   in?: string;
   type?: FilterColorMatrixType;
-  values?: number | Array<number> | string;
+  values?: NumberArray;
```

Comment:
Why did it end up in this PR?

Reply:
Reverted in [31af51b](https://github.com/software-mansion/react-native-svg/pull/2392/commits/31af51bf062a2054b5082d2693829a2f7b3e46a4)